presto-main/etc/config.properties

```diff
@@ -27,6 +27,7 @@ query.client.timeout=5m
 query.min-expire-age=30m
 
 plugin.bundles=\
+  ../presto-resource-group-managers/pom.xml,\
```

Comment:
nitish nit: this list is "semi-sorted", maybe we could put it later (or even sort this list).

Reply:
what's the ordering function?